### ISSUE TITLE
uBlock not working on any websites

### ISSUE THREAD

**Issue by silvieb45** [uBlockOrigin/uBlock-issues#2608]
### Prerequisites

- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

Hello!
I'm not sure what happened but, starting midday yesterday, ads have returned on all websites. I cleared cache/cookies/history, turned the blocker on/off, manually blocked off ads, and they still return. 

https://user-images.githubusercontent.com/81541421/233823612-deafcde3-5318-4ad5-a2f1-0e49ae7c6d6b.png

https://user-images.githubusercontent.com/81541421/233823638-85e17a77-5ee1-44e3-ac58-c666d6d64795.png

https://user-images.githubusercontent.com/81541421/233823754-ae308cba-055d-4b06-b1ec-456733e9624a.png

https://user-images.githubusercontent.com/81541421/233823802-b2cd7049-58e8-4ff0-adf3-394f1af21a3f.png

### A specific URL where the issue occurs.

https://www.reddit.com/

### Steps to Reproduce

1. Went on any websites
2. Saw ads

### Expected behavior

Not see any ads

### Actual behavior

I saw ads 

### Configuration

(I'm not really sure what to write, I was hoping on getting advice to see what else I can do since it randomly stopped working. Thank you so much!)

**Comment by mapx-** [comment]
- Reset uBO 
- update all filter lists
- test again

**Comment by gorhill** [comment]
Provide troubleshooting information: go to uBO options => _Support_ pane => _Troubleshooting Information_ => Copy/paste content here.

**Comment by graphixillusion** [comment]
It happened to me too. I've cleared, reset, unistall and reinstall. At first it worked but after a while it stops again. Not working on any website. Even custom blocked elements doesn't work. This is my Troubleshooting Information

```
uBlock Origin: 1.49.0
Chromium: 112
filterset (summary): 
 network: 0
 cosmetic: 0
 scriptlet: 0
 html: 0
listset (total-discarded, last-updated): [none]
filterset (user): [array of 1 redacted]
modifiedUserSettings: 
 advancedUserEnabled: true
 suspendUntilListsAreLoaded: true
modifiedHiddenSettings: [none]
supportStats: 
 allReadyAfter: 906 ms (selfie)
 maxAssetCacheWait: 68 ms
```

**Comment by iam-py-test** [comment]
Your storage is getting corrupted. There is something wrong with your system. Can you create a new profile, install uBlock Origin, and see if the same thing happens? Are you using any

**Comment by gorhill** [comment]
> after a while it stops again

Do you have some sort of anti-virus or "cleaner" that could potentially mess with uBO installation?

**Comment by graphixillusion** [comment]
> > after a while it stops again
> 
> Do you have some sort of anti-virus or "cleaner" that could potentially mess with uBO installation?

Nope. I'm on Windows 10 and i tried several browser. I have the standard windows defender installed and i didn't touch anything. This things started after 1.49 ublock version

**Comment by gorhill** [comment]
Can you install the [dev build](https://github.com/gorhill/uBlock/releases) and see if the same issue occurs? The latest dev build is 1.49.1b4, but it will probably only be available tomorrow in the Chrome Web Store.

**Comment by Lentzeris** [comment]
It happens to me too.

> uBlock Origin: 1.49.0
> Chromium: 112
> filterset (summary): 
>  network: 0
>  cosmetic: 0
>  scriptlet: 0
>  html: 0
> listset (total-discarded, last-updated): [none]
> filterset (user): [array of 57 redacted]
> trustedset: 
>  added: [array of 15 redacted]
>  removed: 
>   about-scheme
>   chrome-extension-scheme
>   chrome-scheme
>   edge-scheme
>   moz-extension-scheme
>   opera-scheme
>   vivaldi-scheme
>   wyciwyg-scheme
> switchRuleset: 
>  added: [array of 4 redacted]
>  removed: 
>   no-large-media: behind-the-scene false
> hostRuleset: 
>  added: [array of 10 redacted]
>  removed: 
>   behind-the-scene * * noop
>   behind-the-scene * image noop
>   behind-the-scene * 3p noop
>   behind-the-scene * inline-script noop
>   behind-the-scene * 1p-script noop
>   behind-the-scene * 3p-script noop
>   behind-the-scene * 3p-frame noop
> modifiedUserSettings: 
>  advancedUserEnabled: true
>  cloudStorageEnabled: true
>  contextMenuEnabled: false
>  suspendUntilListsAreLoaded: true
> modifiedHiddenSettings: 
>  userResourcesLocation: [redacted]
> supportStats: 
>  allReadyAfter: 429 ms (selfie)
>  maxAssetCacheWait: 32 ms

It seems that when the browser starts, the filters are empty:
![image](https://user-images.githubusercontent.com/25702519/233848328-5b0ae818-60ac-474d-ae75-48cc54e5ca98.png)

If I purge caches and force update them, it works normally.

**Comment by iam-py-test** [comment]
> It seems that when the browser starts, the filters are empty:

- are you using any cleaning software?
- have you checked your hard drive?
- have you tried reinstalling uBlock Origin
- Can you try this: https://github.com/uBlockOrigin/uBlock-issues/issues/2608#issuecomment-1519089439

**Comment by graphixillusion** [comment]
> If I purge caches and force update them, it works normally.

Correct, i confirm.

**Comment by gorhill** [comment]
Marking as duplicate of #2604 -- same symptoms.

At this point the fix in 1.49.1b4 is still a guess, and given that the issue occurs rather rarely, I think I will need to publish the guessed fix to find out if this was really the issue.
